code/routers/user/reset.js

```diff
@@ -0,0 +1,203 @@
+const express = require('express');
+const passport = require('passport');
+const router = express.Router();
+
+
+const User = require('../../models/user');
+const resCode = require('../../config/resCode');
+
+var async = require("async");
+var nodemailer = require("nodemailer");
+var crypto = require("crypto");
+
+/*
+  forgot password api
+
+  let's users reset their password and redirect
+  them to the login page
+*/
+
+router.get('/forgot', function(req, res) {
```

Comment:
I don't think we need this anymore since react will now be covering which page we are on. I may be misunderstanding what this is though. It is for redirecting the user the 'forgot' page right? If this is just for redirecting I don't think we need it anymore.

Reply:
I'll test this and remove it if not needed.

Reply:
I've just removed the get method. React does take care of this.